dataprofiler/profilers/categorical_column_profile.py

```diff
@@ -43,6 +43,9 @@ def __init__(self, name: str | None, options: CategoricalOptions = None) -> None
         self.__calculations: dict = {}
         self._filter_properties_w_options(self.__calculations, options)
         self._top_k_categories: int | None = None
+        self.max_sample_size_to_check_stop_condition = None
```

Comment:
I suggest putting comments around this to clarify these are stop profiling conditions